uint8 robot_id
uint8 robot_level
uint16 current_hp
uint16 maximum_hp
uint16 shooter_barrel_cooling_value
uint16 shooter_barrel_heat_limit
uint16 chassis_power_limit
uint8 power_management_gimbal_output
uint8 power_management_chassis_output
uint8 power_management_shooter_output
builtin_interfaces/Time stamp # 时间戳 
